packages/turf-polygon-to-linestring/index.js

```diff
@@ -0,0 +1,52 @@
+var feature = require('@turf/helpers').feature
+/**
+ * Takes a GeoJSON {@link Polygon}Polygon or {@link MultiPolygon}MultiPolygon feature and returns 
+ * it as a GeoJSON  {@link LineString}linestring or {@link MultiLineString}MultiLineString feature.
+ *
+ * @name polygonToLineString
+ * @param {Feature<Polygon> | Feature<MultiPolygon> } a Polygon or MultiPolygon feature
+ * @return {Feature<LineString> | Feature<MultiLinestring> a LineString or MultiLinestring feature
+ * @example
+ * var poly = {"type": "Feature",
+ *            "geometry": {
+ *                "type": "Polygon", 
+ *                "coordinates": [
+ *                	[[-2.275543, 53.464547],
+ *                 	[-2.275543, 53.489271],
+ *                  [-2.215118, 53.489271],
+ *                  [-2.215118, 53.464547],
+ *                  [-2.275543, 53.464547]]
+ *                 ]},
+ *                 "properties": {}; 
+ * 
+ * var newLine = turf.polygonToLineString(poly);
+ * //addToMap
+ * var addToMap = [newLine]
+ */
+ module.exports = function (polygon) {
+ 	var line = JSON.parse(JSON.stringify(polygon))
```

Comment:
This slows down performance significantly, you should not be mutating the input variable.